src/main/kotlin/org/wycliffeassociates/sourceaudio/upload/RequestHandler.kt

```diff
@@ -0,0 +1,76 @@
+package org.wycliffeassociates.sourceaudio.upload
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import io.ktor.http.content.PartData
+import io.ktor.http.content.streamProvider
+import org.wycliffeassociates.sourceaudio.upload.model.FileUploadModel
+import java.io.File
+import java.lang.IllegalArgumentException
+
+class RequestHandler {
+
+    fun handleFormUpload(parts: List<PartData>): String {
+        val paramsMap: MutableMap<String, String> = mutableMapOf()
+        var file = File("./src/main/resources/uploads/")
```

Comment:
I still think this should probably be a property of the class because eventually you will need to save the file and I think it will be best if that file is available to the whole class.

Reply:
Do you mean a data class with only one file property in it?

Reply:
No, I think that the file should be in this class, but it could be declared as a private variable i.e. it would be private val file: File
It would be initialized once the handleFormUpload function is called. Or you could perhaps give the class a constructor that accepts a file as input? That may be over complicating things though